blueflood-http/src/integration-test/java/com/rackspacecloud/blueflood/inputs/handlers/HttpHandlerIntegrationTest.java

```diff
@@ -93,6 +105,45 @@ public void testHttpIngestionHappyCase() throws Exception {
     }
 
     @Test
+    public void testHttpAnnotationsIngestionHappyCase() throws Exception {
+        URIBuilder builder = getMetricsURIBuilder()
+                .setPath("/v2.0/333333/events");
+
+        StringBuilder sb = new StringBuilder();
+        BufferedReader reader = new BufferedReader(new InputStreamReader(new FileInputStream("src/test/resources/events.json")));
+        String curLine = reader.readLine();
+        while (curLine != null) {
+            sb = sb.append(curLine);
+            curLine = reader.readLine();
+        }
+        String json = sb.toString();
+
+        HttpPost post = new HttpPost(builder.build());
+
+        HttpEntity entity  = new StringEntity(json, ContentType.APPLICATION_JSON);
+        post.setEntity(entity);
+
+        HttpResponse response = client.execute(post);
+        Assert.assertEquals(200, response.getStatusLine().getStatusCode());
+
+        //Slight delay in search
```

Comment:
Is this delay necessary? For what its worth, sleeps like these make the tests run slow.